hack/make/dependencies.make

```diff
@@ -65,5 +64,4 @@ $(ISTIOCTL):
 	$(eval ISTIO_TMP := $(shell mktemp -d))
 	cd $(ISTIO_TMP); curl -sSL https://istio.io/downloadIstio | ISTIO_VERSION=$(ISTIOVERSION) sh -
 	cp $(ISTIO_TMP)/istio-$(ISTIOVERSION)/bin/istioctl ${ISTIOCTL}
-	-rm -rf $(TMP)	
-
+	-rm -rf $(TMP)	
```

Comment:
Can probably undo this

Reply:
it was removed before by accident i think, do you we not need it to remove the tmp files i don't think the get deleted automatically

Reply:
I see, i think there was an extra tab and newline here that you removed. My editor is setup to automatically add a newline to the end of any file i edit, it's usually good practise to do that, that's what GH is complaining about.